picard/ui/options/tags.py

```diff
@@ -119,7 +120,7 @@ def save(self):
         config.setting["remove_ape_from_mp3"] = self.ui.remove_ape_from_mp3.isChecked()
         config.setting["remove_id3_from_flac"] = self.ui.remove_id3_from_flac.isChecked()
         config.setting["itunes_compatible_grouping"] = self.ui.itunes_compatible_grouping.isChecked()
-        config.setting["preserved_tags"] = self.ui.preserved_tags.text()
+        config.setting["preserved_tags"] = re.sub(r"[,\s]+$", "", self.ui.preserved_tags.text())
```

Comment:
re.compile somewhere outside?

Reply:
I thought about it, but don't think so. This is a simple regex that doesn't get run frequently and then only once. And I'd rather keep the code here in one place in this case.